frontend/src/metabase/query_builder/actions/parameters.ts

```diff
@@ -0,0 +1,39 @@
+import { createThunkAction } from "metabase/lib/redux";
+import { CardApi } from "metabase/services";
+import { getParameterValuesSearchCache } from "metabase/query_builder/selectors";
+import { Parameter } from "metabase-types/api";
+import { Dispatch, GetState } from "metabase-types/store";
+import Question from "metabase-lib/Question";
+
+interface FetchParameterValuesOpts {
+  question: Question;
+  parameter: Parameter;
+  query?: string;
+}
+
+export const FETCH_QUESTION_PARAMETER_VALUES =
+  "metabase/qb/FETCH_QUESTION_PARAMETER_VALUES";
+
+export const fetchQuestionParameterValues = createThunkAction(
```

Comment:
Here I'm replicating the same logic used for caching parameter values in dashboards.